libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1859,27 +1859,24 @@ RC_Channel *RC_Channels::find_channel_for_option(const RC_Channel::AUX_FUNC opti
 // duplicate_options_exist - returns true if any options are duplicated
 bool RC_Channels::duplicate_options_exist()
 {
-    uint8_t auxsw_option_counts[512] = {};
+    Bitmask<512> used_auxsw_options;
     for (uint8_t i=0; i<NUM_RC_CHANNELS; i++) {
         const RC_Channel *c = channel(i);
         if (c == nullptr) {
             // odd?
             continue;
         }
         const uint16_t option = c->option.get();
-        if (option >= sizeof(auxsw_option_counts)) {
+        if (option == 0) { // MAGIC VALUE! This is AUXSW_DO_NOTHING
```

Comment:
```suggestion
        if (option == (uint16_t)RC_Channel::AUX_FUNC::DO_NOTHING) {
```

Reply:
Done.  No idea why I used a literal zero when I first wrote this.